ucm2/Intel/sof-hda-dsp/dsp.conf

```diff
@@ -0,0 +1,251 @@
+# Control audio processing in SOF
+
+# File paths for controlling SOF processing
+
+Define.SOFVendor "$${sys:devices/virtual/dmi/id/sys_vendor}"
+Define.SOFProduct "$${sys:devices/virtual/dmi/id/product_name}"
+
+If.SOFVendor {
+	Condition {
+		Type String
+		Empty "${var:SOFVendor}"
+	}
+	True.Define.SOFVendor "${sys:devices/virtual/dmi/id/board_vendor}"
+}
+
+If.SOFProduct {
+	Condition {
+		Type String
+		Empty "${var:SOFProduct}"
+	}
+	True.Define.SOFProduct "${sys:devices/virtual/dmi/id/board_name}"
+}
+
+Define {
+	PostMixerAnalogPlaybackIirBytes "not available"
```

Comment:
Just nitpicking. I would use "N/A" here to save some space.

Reply:
Yep, that's better - Fixed.